app/api/custom/orders.py

```diff
@@ -93,3 +102,73 @@ def calculate_amount():
             return UnprocessableEntityError({'source': 'discount-code'}, 'Discount Usage Exceeded').respond()
 
     return jsonify(calculate_order_amount(tickets, discount_code))
+
+
+@order_blueprint.route('/create-order', methods=['POST'])
+@jwt_required
+def create_order():
+    data = request.get_json()
+    tickets = data['tickets']
+    attendee = data['attendee']
+    for attribute in attendee:
+        attendee[attribute.replace('-', '_')] = attendee.pop(attribute)
+    schema = AttendeeSchema()
+    json_api_attendee = {"data": {"attributes": data['attendee'], "type": "attendee"}}
+    result = schema.load(json_api_attendee)
+    if result.errors:
+        return jsonify(result.errors)
+    ticket_ids = [ticket['id'] for ticket in tickets]
+    ticket_info_list = db.session.query(Ticket).filter(Ticket.id.in_(ticket_ids)).all()
```

Comment:
This will allow deleted tickets as well. And as previously instructed, event filtering should be done here itself